python/grass/grassdb/checks.py

```diff
@@ -115,8 +116,40 @@ def get_mapset_owner(mapset_path):
         return None
 
 
-def is_current_mapset_in_demolocation():
-    return gisenv()["LOCATION_NAME"] == "world_latlong_wgs84"
+def is_fallback_session():
+    """Checks if a user encounters a fallback GRASS session.
+
+    Returns True if a user encounters a fallback session.
+    It occurs when a last mapset is not usable and at the same time
+    a user is in a temporary location.
+    """
+    if "LAST_MAPSET_PATH" in gisenv().keys():
```

Comment:
Didn't test, but I believe this can be shortened:
```suggestion
    if "LAST_MAPSET_PATH" in gisenv():
```

Reply:
It does not work like that... I let it as it was.